src/components/AdminManage/Header.tsx

```diff
@@ -6,12 +7,19 @@ import { useRouter } from "next/router";
 //Render header part of admin manager page
 const Header = () => {
   const router = useRouter();
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
```

Comment:
The loggin logic can be reused. You write same code in the Header.tsx and admin-manager.tsx